src/config/settings.py

```diff
@@ -67,6 +72,12 @@ class Settings(metaclass=Singleton):
     sentry_dsn: str
     pool_size: int | None
 
+    enable_api: bool
```

Comment:
I think we can `validators_registration_mode` for checks whether api should be enabled or not. Then we can remove this setting